app/views/appointment_summaries/new.html.erb

```diff
@@ -6,533 +6,497 @@
   <p class="lead">Use this form to enter the visitor's personal details given during the guidance
     appointment and to issue a personalised summary document.</p>
 
-  <div class="display_if_consent">
-    <% if @appointment_summary.errors.any? %>
-      <div class="alert alert-danger" role="alert">
-        <h2 class="alert__heading">There's a problem</h2>
-        <ul>
-          <% @appointment_summary.errors.full_messages.each do |msg| %>
-            <li><%= msg %></li>
-          <% end %>
-        </ul>
-      </div>
-    <% end %>
+  <% if @appointment_summary.errors.any? %>
```

Comment:
This is removing the surrounding DIV which dealt with displaying stuff inside it, if consent (checkbox) was given (ticked).